pages/ceramic.js

```diff
@@ -0,0 +1,23 @@
+import Head from '../components/Head'
+import Header from '../components/Header'
+import { createAccountsDocument, updateAccountsDocument, initOrUpdateContacts } from '../lib/ceramicFunctions'
+
+const ceramic = () => {
```

Comment:
Components are always capitalized `Ceramic`

Reply:
Even when they are pages? I can change it but it's just testing environment. When the integration works I'll delete it